scripts/index.js

```diff
@@ -0,0 +1,14 @@
+const typesVersion = require('@joystream/types/package.json')
+
+const exportedScripts = {}
+
+exportedScripts.example = require('./example.js')
+exportedScripts.exportDataDirectory = require('./export-data-directory.js')
+exportedScripts.injectDataObjects = require('./inject-data-objects.js')
+
+// This scripts uses the joy injected types introduced in @joystream/types v0.12.0
+if (typesVersion.version === '0.12.0') {
```

Comment:
in future it would be good to use `semver` package to do better comparison like version >= 0.12.0 etc..